src/composables/multisigTransactions.ts

```diff
@@ -47,7 +48,20 @@ export function useMultisigTransactions({ store }: UseMultisigTransactionsOption
   }
 
   async function postSpendTx(tx: string, txHash: string) {
-    return postJson(`${activeNetwork.value.multisigBackendUrl}/tx`, { body: { hash: txHash, tx } });
+    const response = await fetch(
+      `${activeNetwork.value.multisigBackendUrl}/tx`,
+      {
+        method: 'post',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ hash: txHash, tx }),
+      },
+    );
```

Comment:
we have such function
, called `postJson`,
 but it is using `fetchJson`
 witch always returning response.json(), because every response should have at least an empty body
, but our backend don't do that

Reply:
I agree with Paweł. So maybe the `postJson` should be tweaked to check the response headers and do the `response.json()` only if the status is `200`?

Reply:
We can do that, but i would prefer to not ask testers to test every other fetch request in this particular PR, because backend or middleware might also have another place where status returned for a fetch transaction is not `200`.